src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -152,35 +112,76 @@
  * Example:
  * </p>
  * <pre>
- * switch (i) {
- *   case 1:
- *     break;
- *   case 2: // Previous case: OK
- *           // This case: violation, last case must have statement that transfer control
- *     i++;
+ * class Example2 {
+ *   public int foo() {
+ *     int i = 0;
+ *     while(i &gt;= 0) {
+ *       switch(i) {
+ *         case 1:
+ *           i++;  // falls-thru
+ *         case 2: // violation
+ *           i++;
+ *           break;
+ *         case 3:
+ *           i++;
+ *           return;
+ *         case 4:
+ *           i++;
+ *           throw new Exception();
+ *         case 5:
+ *           i++;
+ *           continue;
+ *         case 6:
+ *         case 7:
+ *         case 8:
+ *           i++;
+ *         case 11:
+ *           i++; // fall through
```

Comment:
Put this comment on case8.

This line should be violation for such configuration.

Reply:
Done